### PR TITLE
Update the structure

### LINKED ISSUE #4: navigation bar is missing most headlines on main page
At present the navigation bar only shows `MESSAGEix Workshops`, but none of the other high-level headings from the `Table of Content` on the main page. It would be good to add those sections also to the navigation bar by choosing appropriate headings.

### PR DESCRIPTION
**Pull request by LauWien**
This PR updates the structure.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,44 +1,71 @@
...
 Take a look - no need to enroll!
 
 .. toctree::
+   :hidden:
+   :caption: University courses and lectures
    :maxdepth: 1
 
    lectures/osesm-tuwien/index
 
+- :doc:`lectures/osesm-tuwien/index`
```
Comment (danielhuppmann) [inline index.rst]:
```suggestion
- :doc:`lectures/osesm-tuwien/index` by Daniel Huppmann (2019-ongoing)
```

**Review by danielhuppmann — APPROVED** [review]
Very nice idea to have both a hidden TOC and an explicit `:doc:...` call, because this gives a lot more flexibility in adding text below the TOC caption and also more info on the overview page than what should be in the title (h1) of the sub-page.

I made one suggestion to use that flexibility right away... ;-)

One further suggestion to make the duplicate TOC-and-doc strategy more obvious and prevent that users add it only or the other - add a note in the "How to add new content" section of the Readme...

**Review by danielhuppmann — APPROVED** [review]
Great idea to add an explicit example!

**Comment by LauWien** [comment]
> 
> 
> Great idea to add an explicit example!

:)

**Pull request merged**